webots_ros2_turtlebot/package.xml

```diff
@@ -15,6 +15,8 @@
   <exec_depend>builtin_interfaces</exec_depend>
   <exec_depend>webots_ros2_driver</exec_depend>
```

Comment:
```suggestion
  <exec_depend>webots_ros2_core</exec_depend>
  <exec_depend>webots_ros2_driver</exec_depend>
```
Since it is used here: https://github.com/cyberbotics/webots_ros2/pull/221/files#diff-4ea1ac12cdfd71571f0a857c8aaa616e8a48b0eda65f8cde8693ae58f97999f1R28